usr/src/contrib/libfdt/libfdt_env.h

```diff
@@ -7,17 +7,34 @@
  * Copyright 2012 Kim Phillips, Freescale Semiconductor.
  */
 
+#if defined(_STANDALONE)
+#include <stand.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <limits.h>
+#else
 #include <sys/inttypes.h>
 #include <sys/null.h>
 #include <sys/types.h>
 #include <sys/stdbool.h>
+#endif
 
 #if defined(_KERNEL) && !defined(_BOOT)
 #include <sys/ddi.h>
 #include <sys/sunddi.h>
 #else
+#if !defined(_STANDALONE) && !defined(_BOOT)
 #include <stdlib.h>
 #include <string.h>
```

Comment:
I half feel like these could be made more readable with #elif, but I'm not actually certain

Reply:
Reworked in latest push.